onepixel.production.env

```diff
@@ -1,2 +1,3 @@
 ADMIN_SITE_HOST=onepixel.link
-MAIN_SITE_HOST=1px.li
\ No newline at end of file
+MAIN_SITE_HOST=1px.li
+URL_GEN_MAX_RETRIES=1000
```

Comment:
production retries should not be 1000 either, 10 is a better limit 
commenting below, why the error message is not good

Reply:
Changed it to 10